tools/ci_build/github/azure-pipelines/templates/orttraining-linux-gpu-ortmodule-test-ci-pipeline.yml

```diff
@@ -18,22 +18,10 @@ steps:
   displayName: 'Mount hf-models-cache'
   condition: succeededOrFailed()
 
```

Comment:
1. Please give me a link of your pipeline.
2. Use this template with cache: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/templates/get-docker-image-steps.yml

Reply:
3. Do not have "-p password" things in your bash file, in case the password gets printed out. Normally ADO could help us mask passwords, but we shouldn't rely on that. Use env var instead of command line parameters.

Reply:
1- We are using an existing pipeline : https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=84&_a=summary
2- the pipeline uses the template with cache mentioned.

Reply:
> 1. Please give me a link of your pipeline.
> 2. Use this template with cache: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/templates/get-docker-image-steps.yml

1- We are using an existing pipeline : https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=84&_a=summary
2- the pipeline uses the template with cache mentioned.



> 3. Do not have "-p password" things in your bash file, in case the password gets printed out. Normally ADO could help us mask passwords, but we shouldn't rely on that. Use env var instead of command line parameters.

3- it is already an environment variable, and it is defined in the variable group in the pipeline definition.

Reply:
The link you gave me is "orttraining-linux-gpu-ci-pipeline". It is a for pull requests. So you are modifying this pipeline?

Reply:
> The link you gave me is "orttraining-linux-gpu-ci-pipeline". It is a for pull requests. So you are modifying this pipeline?

Yes, we are modifying the pipeline so that we can have scheduled build with the nightly pytorch version.

Reply:
You may need to manually copy the nightly version to an internal safe place, then install pytorch from there. Is it acceptable?

Reply:
@baijumeswani, @AdamLouly: Can we use PTCA image with pytorch nightly?

Reply:
We could potentially do this, though we would need to check if the PTCA image has support for dev tools to build ort with cuda. This would most likely involve more work, but if this is security-wise safer, we can do the work.
@snnn what is the concern with downloading the pip package on every nightly run?

Reply:
No.
I think PTCA is better. It is what we ship. It is closer to the real production environment. 
But if you don't have time to do it now and want to stick with the current plan, it's fine too. 
Please have a discussion with Ashwini and let me know what you think, then I will approve this PR.